packages/vitest/src/node/test-run.ts

```diff
@@ -74,12 +74,12 @@ export class TestRun {
       await Promise.all([
         this.vitest.report('onTestRunEnd', modules, [...errors] as SerializedError[], state),
```

Comment:
shouldn't times be passed down somehow to new hooks? At one point, `onFinished` won't be part of the API